src/mapping/lending-pool-address-provider/v3.ts

```diff
@@ -47,6 +47,14 @@ function genericAddressProviderUpdate(
   }
   pool.lastUpdateTimestamp = event.block.timestamp.toI32();
   pool.save();
+  if (component == 'poolConfigurator') {
```

Comment:
added the configurator-> pool mapping here as i need it to set the fee on the pool based on events emitted on configurator.